spec/controllers/clients_controller_spec.rb

```diff
@@ -115,33 +115,43 @@
       end
 
       context "with existing contact history" do
+        let(:twilio_status) { nil }
         let!(:expected_contact_history) do
           [
             create(:outgoing_text_message, body: "Your tax return is great", sent_at: DateTime.new(2020, 1, 1, 0, 0, 1), client: client, twilio_status: twilio_status),
             create(:incoming_text_message, body: "Thx appreciate yr gratitude", received_at: DateTime.new(2020, 1, 1, 0, 0, 2), client: client),
+            create(:outgoing_email, body: "We are really excited to work with you", sent_at: DateTime.new(2020, 1, 1, 0, 0, 3), client: client),
           ]
```

Comment:
Note for a way we might improve these expectations, possibly create records in a different order from their expected listing and date. Perhaps create in reverse order and then reverse list?